<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>KHotKeys::Dbus_widget_ui</class>
  <widget class="QWidget" name="Dbus_widget_ui" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>576</width>
        <height>480</height>
      </rect>
    </property>
    <layout class="QVBoxLayout" >
      <property name="margin" >
        <number>11</number>
      </property>
      <property name="spacing" >
        <number>6</number>
      </property>
      <item>
        <widget class="QLabel" name="remote_app_label" >
          <property name="text" >
            <string>Remote &amp;application:</string>
          </property>
          <property name="buddy" stdset="0" >
            <cstring>remote_app_lineedit</cstring>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLineEdit" name="remote_app_lineedit" />
      </item>
      <item>
        <spacer name="Spacer1" >
          <property name="sizeHint" >
            <size>
              <width>0</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
      <item>
        <widget class="QLabel" name="remote_object_label" >
          <property name="text" >
            <string>Remote &amp;object:</string>
          </property>
          <property name="buddy" stdset="0" >
            <cstring>remote_object_lineedit</cstring>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLineEdit" name="remote_object_lineedit" />
      </item>
      <item>
        <spacer name="Spacer2" >
          <property name="sizeHint" >
            <size>
              <width>0</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
      <item>
        <widget class="QLabel" name="called_function_label" >
          <property name="text" >
            <string>Called &amp;function:</string>
          </property>
          <property name="buddy" stdset="0" >
            <cstring>called_function_lineedit</cstring>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLineEdit" name="called_function_lineedit" />
      </item>
      <item>
        <spacer name="Spacer3" >
          <property name="sizeHint" >
            <size>
              <width>0</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
      <item>
        <widget class="QLabel" name="arguments_label" >
          <property name="text" >
            <string>Arguments:</string>
          </property>
          <property name="buddy" stdset="0" >
            <cstring>arguments_lineedit</cstring>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLineEdit" name="arguments_lineedit" />
      </item>
      <item>
        <spacer name="Spacer4" >
          <property name="sizeHint" >
            <size>
              <width>0</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
      <item>
        <layout class="QHBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <property name="spacing" >
            <number>6</number>
          </property>
          <item>
            <spacer name="Spacer1_2" >
              <property name="sizeHint" >
                <size>
                  <width>20</width>
                  <height>0</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Qt::Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="try_button" >
              <property name="text" >
                <string>&amp;Try</string>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="Spacer5" >
              <property name="sizeHint" >
                <size>
                  <width>20</width>
                  <height>0</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Qt::Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="PushButton1" >
              <property name="text" >
                <string>Run &amp;D-Bus browser</string>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="Spacer6" >
              <property name="sizeHint" >
                <size>
                  <width>20</width>
                  <height>0</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Qt::Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <layoutfunction spacing="KDialog::spacingHint" margin="KDialog::marginHint" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <tabstops>
    <tabstop>remote_app_lineedit</tabstop>
    <tabstop>remote_object_lineedit</tabstop>
    <tabstop>called_function_lineedit</tabstop>
    <tabstop>arguments_lineedit</tabstop>
    <tabstop>try_button</tabstop>
    <tabstop>PushButton1</tabstop>
  </tabstops>
  <includes>
    <include location="global" >kdialog.h</include>
  </includes>
</ui>
